packages/help-center/src/components/help-center-contact-page.tsx

```diff
@@ -186,18 +184,20 @@ export const HelpCenterContactPage: FC< HelpCenterContactPageProps > = ( {
 
 			const escapedWapuuChatId = encodeURIComponent( wapuuChatId || '' );
 
-			openChatWidget( {
+			openZendeskWidget( {
 				aiChatId: escapedWapuuChatId,
 				siteUrl: site?.URL,
 				onError: () => setHasSubmittingError( true ),
-				// Reset Odie chat after passing to support
-				onSuccess: () => setWapuuChatId( null ),
```

Comment:
Maybe we need to keep the `setWapuuChatId( null )`  call?

Reply:
This was a request to be removed here ⇢ https://docs.google.com/spreadsheets/d/1URiJTt4CC5bC_7-m3UolwrhcJP9ZbaLB7uC-AWC5DaI/edit?gid=1972884515#gid=1972884515&range=40:40